Documentation/ceph-upgrade.md

```diff
@@ -5,53 +5,53 @@ indent: true
 
-With the release of Rook 0.9, significant progress has been made toward the goal of incorporating
-an automated upgrade solution into the Rook operator. While significant, this has merely lain the
-foundation for automated upgrade support, and we will continue to be open to community feedback for
-further improving and refining upgrade automation.
+With the release of Rook 1.0, upgrades for both the operator and for Ceph are nearly entirely
+automated save for where Rook's permissions need to be explicitly updated by an admin. Achieving the
+level of upgrade automation has been refined by community feedback, and we will always be open to
+further feedback for improving automation and improving Rook.
 
 We welcome feedback and opening issues!
 
 
 ## Supported Versions
-The supported version for this upgrade guide is **from a 0.8 release to a 0.9 release**.
-Build-to-build upgrades are not guaranteed to work. This guide is to perform upgrades only between
-the official releases.
+The supported version for this upgrade guide is **from a 0.9 release to a 1.0 release**. This guide
+supports upgrades only between the official releases. Container images for official releases are
+hosted on the Docker Hub project [rook/ceph](https://hub.docker.com/r/rook/ceph/tags) and are
+denoted by tags `vX.Y.Z` or `vX.Y.Z-stable`; these two formats are synonymous.
 
 For a guide to upgrade previous versions of Rook, please refer to the version of documentation for
 those releases.
+- [Upgrade 0.8 to 0.9](https://rook.io/docs/rook/v0.9/upgrade.html)
 - [Upgrade 0.7 to 0.8](https://rook.io/docs/rook/v0.8/upgrade.html)
 - [Upgrade 0.6 to 0.7](https://rook.io/docs/rook/v0.7/upgrade.html)
 - [Upgrade 0.5 to 0.6](https://rook.io/docs/rook/v0.6/upgrade.html)
 
-### Patch Release Upgrades
-One of the goals of the 0.9 release is that patch releases are able to be automated completely by
-the Rook operator. It is intended that upgrades from one patch release to another are as simple as
-updating the image of the Rook operator. For example, when Rook v0.9.1 is released, the process
-should be as simple as running the following:
-```
-kubectl -n rook-ceph-system set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v0.9.x
-```
-
 
 ## Considerations
 With this upgrade guide, there are a few notes to consider:
 * **WARNING:** Upgrading a Rook cluster is not without risk. There may be unexpected issues or
   obstacles that damage the integrity and health of your storage cluster, including data loss. Only
   proceed with this guide if you are comfortable with that.
 * The Rook cluster's storage may be unavailable for short periods during the upgrade process for
-  both Rook operator updates and for Ceph daemon updates.
-* Rook is able to handle a great deal of the upgrade on its own, but manual steps are still
-  necessary. This is in part to reduce the Kubernetes privileges required by the Rook operator.
+  both Rook operator updates and for Ceph version updates.
```

Comment:
With #2901 this should improve or even hopefully eliminate downtime.

Reply:
Given that it's not in the v1.0 project, I'm not sure if I can rely on that being the case.

Reply:
#2901 does contain a fix for #2973, so we either need to finish off #2901 or split it out to get the minimal fix for #2973. Just wanted to point out that if #2901 goes in, we could edit this statement.

Reply:
I commented more here, but I think due to time constraints looking at the v1.0 release, a minimal fijx for #2973 would be best. https://github.com/rook/rook/pull/2901#issuecomment-485930988